index.js

```diff
@@ -16,3 +19,26 @@ mobileLinks.forEach((link) => {
     mobileMenu.classList.toggle('active');
   });
 });
+
+form.addEventListener('submit', (e) => {
+  if (email.value !== email.value.toLowerCase()) {
+    e.preventDefault();
+    if (!email.validity) {
```

Comment:
Well done so far 💯 but this condition is blocking your error from showing, I suggest removing that if to make the error show when the input email isn't correct ❌